slackcli/utils.py

```diff
@@ -62,6 +62,13 @@ def print_messages(source_name, count=20):
     for message in messages[::-1]:
         print(format_message(source_name, message))
 
+def format_outgoing_message(message):
+    usernames = re.search('@([^\s]+)', message)
+
+    if usernames:
+        message = re.sub(r'@([^\s]+)', names.get_user_id(r'<@1>'), message)
```

Comment:
1. It seems to me that it is unnecessary to repeat that regular expression here. Instead, there must be an easier way to search and replace in one shot, although I don't remember how to do it on top of my head.
2. Also, that `names.get_user_id(r'<@1>')` seems fishy. 
3. We will have an issue when the user is not found or does not exist. (userid will be "None")